.gitignore

```diff
@@ -7,3 +7,4 @@ coverage/
 .DS_Store
 npm-debug.log
 yarn-error.log
+package-lock.json
```

Comment:
Probably don't need this line as we should probably use yarn. Since it seems that there's no yarn.lock change here for these additions, we should run `yarn` before checking in.

Reply:
@KevinGrandon I use `npm` , so when developing, there'll be `package-lock.json`.